packages/fuel-gauge/src/advanced-logging.test.ts

```diff
@@ -239,14 +288,35 @@ describe('Advanced Logging', () => {
       amount,
     ];
 
-    beforeAll(async () => {
-      wallet = await generateTestWallet(provider, [[300_000, baseAssetId]]);
-    });
+    beforeAll(async () => {});
 
     it('when using InvocationScope', async () => {
-      const script = new Script(binHexlified, abiContents, wallet);
+      using launched = await launchTestNode({
+        contractsConfigs: [
+          { deployer: AdvancedLoggingAbi__factory, bytecode: AdvancedLoggingAbiHex },
+          {
+            deployer: AdvancedLoggingOtherContractAbi__factory,
+            bytecode: AdvancedLoggingOtherContractAbiHex,
+          },
+        ],
+        walletsConfig: {
+          amountPerCoin: 300_000,
+        },
```

Comment:
```suggestion
        ]
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.